README.md

```diff
@@ -1,47 +1,43 @@
 # lock_and_cache
 
-Lock and cache using redis!
+Lock and cache using Redis!
 
 Most caching libraries don't do locking, meaning that >1 process can be calculating a cached value at the same time. Since you presumably cache things because they cost CPU, database reads, or money, doesn't it make sense to lock while caching?
 
 ## Quickstart
 
-```js
-const lnc = require('lock_and_cache')
+```ts
+import { LockAndCache } from "../lib";
 
-let cache = lock_and_cache.LockAndCache()
+const cache = new LockAndCache();
 
-// standalone mode
-function getStockQuote (symbol) {
-  return cache.get(['stock', symbol], 60, async function () {
-    // fetch stock price from remote source, cache for one minute
-    // calling this multiple times in parallel will only run it once
-  })
+// Standalone mode.
+function getStockQuote(symbol: string) {
+  return cache.get(["stock", symbol], { ttl: 60 }, async () => {
+    // Fetch stock price from remote source, cache for one minute.
+    //
+    // Calling this multiple times in parallel will only run it once.
+    return 100;
+  });
 }
 
-// wrap mode
-const getStockQuote = cache.wrap(60, async function stock (symbol) {
-  // fetch stock price from remote source, cache for one minute
-  // calling this multiple times in parallel will only run it once
-  // the cache key is based on the function name and arguments
-})
-
-// custom options w immediate closing
-// options are passed thru to cache manager which passes thru to backends
-const customCache = new lnc.LockAndCache({
-  caches: cache.tieredCache({...memOpts}, {...redisOpts})
-})
-closing(customCache, async function (cache) {
-  const value = cache.get(...)
-  const wrapped = cache.wrap(...)
-  cache.del(key)
-  ...
-})
+// Wrap mode
+async function stockQuoteHelper(symbol: string) {
+  // Fetch stock price from remote source, cache for one minute.
+  //
+  // Calling this multiple times in parallel will only run it once the cache key
+  // is based on the function name and arguments.
+  return 100;
+}
```

Comment:
now that you removed multi-level caching, i think you can do `cache.wrap` on the first function in the README, you don't need a second one

Reply:
`get` and `wrap` have nothing to do with multi-leveling caching. They're the two main APIs that the library has always supported, originally as `cache(...)` and `cache.wrap(...)`. But I've removed the `get` example from the `README`, though I left it in the example program.

Reply:
Actually, `wrap` can be fairly awkward in certain circumstances, so I really do think that it's important to have `get` as a supported API.